### ISSUE TITLE
Can this be used to make an interactive chess course with quizzes?

### ISSUE THREAD

**Issue by ottomatek** [yo35/rpb-chessboard#239]
I'm currently looking for a way to make chess courses for my website. I'm hoping they can be made into a similar hand-on, interactive learning experience similar to that of chessable or an interactive lichess studies. My first thought was somehow pulling an interactive study pgn off of lichess to upload with this plugin. But then I thought maybe those specific interactive features would be limited to the lichess system and tools, and that the only things that would be uploaded are the actual moves being shown, not the interactive feature. Now I'm wondering if there is some way to somehow pair this plugin with a course plugin or something of that nature. I haven't made the site yet, so it's kind of hard to dive into this a little deeper. Right now, I'm just trying to plan out what's possible, so my question is can I do this and how? Thanks.

**Comment by yo35** [comment]
Hi,
What is possible to achieve with RPB Chessboard is presented in https://rpb-chessboard.yo35.org/ . I don't know whether this corresponds to what you want or not.

Regarding quizzes: this topic has been discussed in other issues, in particular #17 and #58. To cut a long story short: there is no quizz embedded in RPB Chessboard, but it may be possible to implement training puzzles using RPB Chessboard together with some general purpose quizz plugins for WordPress. Examples of such plugins are given in #58. This is just a suggestion, I've not tried that by myself.

**Comment by ottomatek** [comment]
Thanks for the quick response! Yeah, I did see those before posting this, but I'm not sure it's what I'm looking for. I'm trying to set something up that is interactive for the user. For example, step one would be guiding the user through an opening with explanations, arrows, circles, etc. Step 2 would then allow the user to go through the same process but without the explanations, arrows, and circles like a quiz. I'm just not sure how combining the quiz plugin with your chessboard plugin would play out or how it's accomplished. Most of the quiz plugins I have seen do things like multiple choice, so it doesn't seem fitting to what I'm trying to do. I've been looking everywhere but there doesn't seem to be any info out there regarding this specific scenario.

**Comment by yo35** [comment]
> Your chess diagram example, here, gives the example of white to move and mate in 2. Is this something that the reader/user can actively perform, themselves, on the chessboard like a puzzle?

No, the reader/user cannot interact with the diagram. From the reader's point of view, the diagram behaves indeed as a non-interactive picture.
